### DISCUSSION TITLE
基于ppocrv4 finetune的文档能否更新一下

### DISCUSSION THREAD

**Discussion by spencergg** [PaddlePaddle/PaddleOCR#12614] [Q&A]
https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/finetune.md
https://aistudio.baidu.com/modelsdetail/286/example?modelId=286

当前finetune的介绍都是基于v3的，v4做了不少变化，能否更新一下呢？辛苦~

**Comment by youde2000** [comment]
插个眼，我最近也在关注这个

**Comment by QQQTAO** [comment]
> https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/doc/doc_ch/finetune.md https://aistudio.baidu.com/modelsdetail/286/example?modelId=286
> 
> 当前finetune的介绍都是基于v3的，v4做了不少变化，能否更新一下呢？辛苦~

v4识别的训练配置文件和训练代码不一致都没有解决，现在这个项目都没有人解决问题了

**Comment by zz1988** [comment]
是的，期待更新

**Comment by iPengXPro** [comment]
有人跑通ppocrv4 finetune了吗？能否指导一下？谢谢！
